torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2689,12 +2691,31 @@ at::Tensor XLANativeFunctions::select_scatter(const at::Tensor& base,
                                               const at::Tensor& mutated_view,
                                               int64_t dim, int64_t index) {
   TORCH_LAZY_FN_COUNTER("xla::");
-  auto base_ = bridge::GetXlaTensor(base);
-  auto mutated_view_ = bridge::GetXlaTensor(mutated_view);
-  auto base_clone = tensor_methods::clone(base_);
-  auto base_clone_slice = tensor_methods::select(base_clone, dim, index);
-  tensor_methods::copy_(base_clone_slice, mutated_view_);
-  return bridge::AtenFromXlaTensor(base_clone);
+  auto base_tensor = bridge::GetXlaTensor(base);
+  auto mutated_view_tensor = bridge::GetXlaTensor(mutated_view);
+  auto base_tensor_shape = base_tensor->shape();
+  auto mutated_view_tensor_shape = mutated_view_tensor->shape();
+  auto common_device = torch_xla::bridge::GetXlaDevice(base);
+
+  torch::lazy::NodePtr mutated_view_tensor_reshaped_node =
+      torch::lazy::MakeNode<Resize>(
+          mutated_view_tensor->GetIrValue(),
+          torch::lazy::ToVector<int64_t>(base_tensor_shape.get().dimensions()));
+  auto mutated_view_tensor_reshaped = torch_xla::XLATensor::Create(
```

Comment:
You probably don't need this intermediate tensor. However, it might not solve your problem.